apps/backoffice-v2/src/common/hooks/useFilterId/useFilterId.tsx

```diff
@@ -1,6 +1,6 @@
 import { useSearchParams } from 'react-router-dom';
 
-export const useFilterId = (): string | undefined => {
+export const useFilterId = (): string | null => {
```

Comment:
Return types tend to be a moving target. If the type is wrong we'd get the type errors on the consuming side either way.
```suggestion
export const useFilterId = () => {
```